app/code/core/Mage/Adminhtml/Block/Rating/Grid.php

```diff
@@ -30,8 +30,8 @@ public function __construct()
     {
         parent::__construct();
         $this->setId('ratingsGrid');
-        $this->setDefaultSort('rating_code');
-        $this->setDefaultDir('ASC');
+        $this->setDefaultSort('rating_id');
+        $this->setDefaultDir('desc');
```

Comment:
Seems opinion based. I'd prefer sort by name instead of ID.

Reply:
oh sorry, yes, I forgot to remove that for the pr, I will update